tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1445,7 +1445,7 @@ def _impl(ctx):
             unfiltered_compile_flags_feature,
             treat_warnings_as_errors_feature,
             archive_param_file_feature,
-        ] + layering_check_features(ctx.attr.compiler)
+        ]
```

Comment:
It looks like this code defines the layering_check feature but does not enable it.  Do you know where the feature is actually enabled?  One option is to find where the feature is enabled and disabled it there.  However, I see that this code does enable the module_maps feature which does have overhead and is not useful without layering check, so perhaps this is ok.

Reply:
I don't see off hand where it gets enabled 🤔 but i guess the question is do we want folks to be able to enable it at all right now on macOS even if it was opt in? I haven't ever tested this externally on any platform so I'm not sure how well it works as is

Reply:
Ideally it would be nice to leave the feature in that state: disabled but functional.  We do have it enabled internally by default, but we had to individually opt out lots of rules and packages.

It is unsettling that we don't know how this gets enabled, but I don't have time to hunt that down right now.  I'll approve it but make a note that we might want to revisit this in the future.